Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,17 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+{- | These are shorthand for merging lists of patterns with @#@, @|*|@, @|+|@,
```

Comment:
I think we can go with these for now.

Just tried them for a few hours and the improvement from having _any_ operator to work on a a list of `ParamPattern` is huge!

These however are especially handy since they are easily memorable.

We can change to different _names_ later on, but I'd rather have these shortcuts **in** 0.9 than not.

What would be interesting to know is, whether these _helpers_ actually improve usability for others or rather add confusion (even more symbols!)

Maybe this however depends on the prominence of these shortcuts within our docs (right now this would only show on hackage, however, I just read, people get their information about how Tidal works from exactly there!)